pkg/appfile/parser.go

```diff
@@ -93,6 +95,8 @@ type Trait struct {
 	Template           string
 	HealthCheckPolicy  string
 	CustomStatusFormat string
+
+	FullTemplate *util.Template
```

Comment:
why do we need any of the above when we have the fullTemplate when all of them are copied from the fullTemplate.